schema/snapcraft.yaml

```diff
@@ -137,26 +137,28 @@ properties:
             description: plugin name
           source:
             type: string
-          source-type:
-            type: string
+            default: '.'
           source-branch:
             type: string
-          source-tag:
-            type: string
+            default: ''
           source-commit:
             type: string
+            default: ''
+          source-depth:
+            type: integer
+            default: 0
           source-subdir:
             type: string
+            default: ''
+          source-tag:
+            type: string
+            default: ''
+          source-type:
+            type: string
+            default: ''
           disable-parallel:
               type: boolean
               default: false
-          stage-packages:
```

Comment:
It was duplicated, actually-- expand the file below, you'll see it again next to `build-packages`.